parser/parser_test.go

```diff
@@ -163,53 +163,53 @@ func (s *parserMultiIncludeTestSuite) TestParserMultiInclude_FromMultiFile() {
 	s.Equal(expected.Len(), n)
 }
 
-// Templates test suite
-type parserTemplateTestSuite struct {
+// Definitions test suite
+type parserDefinitionTestSuite struct {
 	suite.Suite
 	ctx    *processors.Context
 	reader io.Reader
 }
 
-func (s *parserTemplateTestSuite) SetupSuite() {
+func (s *parserDefinitionTestSuite) SetupSuite() {
 	s.ctx = processors.NewContext(os.TempDir())
-	s.reader = strings.NewReader("##!> template this-is-a-text [a-zA-J]+8\n" +
-		"##!> template this-is-another-text [0-9](pine|apple)\n" +
-		"{{this-is-a-text}} to see if templates work.\n" +
+	s.reader = strings.NewReader("##!> definition this-is-a-text [a-zA-J]+8\n" +
+		"##!> definition this-is-another-text [0-9](pine|apple)\n" +
```

Comment:
```suggestion
		"##!> define this-is-another-text [0-9](pine|apple)\n" +
```

I see what you did there 🍍